### ISSUE TITLE
Formating of lists

### ISSUE THREAD

**Issue by afulki** [mudasobwa/markright#12]
In your test for LI you have the following:

```quote
  Hello, world! List here:
  - item 1
  - item 2
  - item 3

  Afterparty.
```

When converted to AST, I get: 

```
{:article, %{},
 [
   {:p, %{},
    [
      "Hello, world! List here:\r\n\r",
      {:ul, %{},
       [
         {:li, %{}, "item 1"},
         {:li, %{}, "item 2"},
         {:li, %{}, "item 3\r\n\r\nAfterparty."}
       ]}
    ]}
 ]}
```

Notice the "afterparty is being treated as part of item3 not as a separate paragraph.

**Comment by am-kantox** [comment]
@afulki 

I honestly did not look in this code for almost 4 years. I have a lazy plan to rewrite it from scratch to support tables etc.

Meanwhile, I will take a look this weekend to check why it fails to understand the list above. Sorry for the inconvenience.

**Comment by afulki** [comment]
Thank you, much appreciated!

**Comment by am-kantox** [comment]
@afulki 

Well, it took way longer than I promised, sorry for that. I have upgraded all the ancient versions and released `v0.6.0`. The issue was most likely induced by legacy version of `xml_builder`. Now everything is green.

```elixir
iex|1 ▶ input = """
...|1 ▶   Hello, world! List here:
...|1 ▶   - item 1
...|1 ▶   - item 2
...|1 ▶   - item 3
...|1 ▶ 
...|1 ▶   Afterparty.
...|1 ▶   """
"Hello, world! List here:\n- item 1\n- item 2\n- item 3\n\nAfterparty.\n"
iex|2 ▶ Markright.to_ast(input)
{:article, %{},
 [
   {:p, %{},
    [
      "Hello, world! List here:",
      {:ul, %{},
       [{:li, %{}, "item 1"}, {:li, %{}, "item 2"}, {:li, %{}, " item 3"}]},
      "Afterparty.\n"
    ]}
 ]}
```
